### DISCUSSION TITLE
Synchronization between the selection of a variable in the desassembly Window and the Stack variables

### DISCUSSION THREAD

**Discussion by ethergrouik** [Vector35/binaryninja-api#3862] [Ideas] [answered]
Hello,
Would it be possible to add a feature that would synchronize the selection of a variable in the disassembly part with the Stack variables? Unless there is an existing trick that I don't know ;)

Thanks in advance

**Comment by plafosse** [comment, marked as answer]
Yes it would be possible it just something we haven't implemented yet, as there isn't a way for the Sidebar widget to get a callback when the token changes. This is captured in this issues https://github.com/Vector35/binaryninja-api/issues/2712

**Comment by xusheng6** [comment]
Related to https://github.com/Vector35/binaryninja-api/issues/5269 and https://github.com/Vector35/binaryninja-api/issues/2712
